File: src/Roguelike/Core/Data/Combatant/HeroData.cs
using System.Collections.Generic;

namespace Roguelike.Data
{


    public class HeroData : CombatantData
    {


        public int StartingGold { get; set; }


        public int StartingMana { get; set; }


        public int StartingHandSize { get; set; }


        public List<string> StartingDeckCardIds { get; set; } = new List<string>();


        public string StartingRelicId { get; set; }
    }
}

File: python_roguelike/data/combatant/hero_data.py
from dataclasses import dataclass, field
from typing import List
from .combatant_data import CombatantData


@dataclass
class HeroData(CombatantData):
    starting_gold: int = 0
    starting_mana: int = 3
    starting_hand_size: int = 5
    starting_deck_card_ids: List[str] = field(default_factory=list)
    starting_relic_id: str = ""
